### DISCUSSION TITLE
How to fetch a recorded audio to transcribe using transcribe () method?

### DISCUSSION THREAD

**Discussion by jayantabas1** [openai/whisper#1789] [Q&A]
I am developing a flask application , where I have incorporated whisper tiny model. I am recording an audio using js and defining backend route to transcibe the audio. Currently the problem is I have to save the audio locally first then by providing the path/<audio_name> to transcribe() method I am able to trabscribe the audio. Is there any other method to do that without saving audio locally? Need a workaround to do that.
 def translate():
    audio_file = request.files['audio']
   ----------------------------
------------------------------
    result = model.transcribe(audio_file ).
Since the audio_file  is a FileStorage type, How to make it transcribable so that transcribe method accept it as parameter?

**Comment by glangford** [comment]
> Since the audio_file is a FileStorage type, How to make it transcribable so that transcribe method accept it as parameter?

Since the FileStorage type is a wrapper over files, can you pass `audio_file.filename` to `transcribe`

(not a Flask user, just going by documentation)
